common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala

```diff
@@ -0,0 +1,113 @@
+package com.gu.mediaservice.lib.config
+
+object UsageRightsConfig {
+
+  // Note: we filter exclusively on matching source, rather than combining credit=Getty and source=X
+  // this is assumed to be good enough as it's unlikely other provider will use the same source.
+  val payGettySourceList = List(
+    "ASAblanca",
```

Comment:
This has been in Config since the start.
We could do - but what would probably be best is to have it in a store, as, eventually, this will become editable.